.travis.yml

```diff
@@ -0,0 +1,29 @@
+dist: trusty
+sudo: required
+services:
+- docker
+notifications:
+  email: false
+env:
+  global:
+  - secure: fpFAa2wHk4EDfPl5cfDtVsxZtdXOiR3NuZaTehoeXjokF5KpSv6FbTSAk/N5jnpvo5zMYhoNV06RK+AFsSCJ478gfcdbaqAuWd+Zpwqx/R7P6M2+LpKAixTlfmB4CjjG3M0jiovjCI1HN7nca3DKyb+t0hbVe0vZxkbyNdrMqufWoCPZP8Js9dACVtb1OY1uHPhQFgpDCP37VFblg2OMN5Exg5A6EE03VcgV6cMInYASA9JttRxaP71ytlTNwc1b2fa/HCM8WCXN1cksnipvHGRqh99oWMRuS9o4OU8+ckD4pJJMxOeT97I/uaTVAy3TE8m/18jQkzm18yVtlyW9A2OwQVMGbNk5bJaT9AOXtO9ViGdf2WzmkelJZQIA4mnU8G0P1oXs3qO4u1J079n/vlb/8hc4yoaVYvuvqx2XMf1kZ3M30jNnH8yRc6Q0+cw/B7GzbOHlTR6wvYRbWHvfuI8ZOLS8VWRjxtgB2fhKh2qP4lmv6TPKyTFAlMmcCTzKQTQ+DKycdF+Q4pSc1XVIDvVKBq57qSos7VRVhSJxkNLHIZzul8efGw/o/KIhVecDtgVPPP0BByZg1NZYKa+b6t9Y1149R1CrBJn5a1k//grPQ0je/mbS2AfQiR4bLK7dj8Y0TH11rvvqEALkA5gNSUyFg+zp9IFO3i2Ulg0ULEA=
+  - secure: Mugr2Z23peIoB1cRN5VQpuOtCOkjpd9rLNrtN9yQ/fLoLam6YNJjfXg0QG/u5UZD0LTD18urTkv68ltNxYVzdHAaxs2Ml4tEbnEgSNk321P46JK9YUo5lJdvAIgIuZeymucdSpKQV5MN8R5dUmdNlXZSsqtZFWxE00C7vKivhleBCEEPUD5QDZU2BytDdyK7NF2paCYSDdCUViIRF+xQFLdPlAnrH9NaLtZ3bGYQOt9BIksnj4lIEs0h+h21u8IU+fzUVXP386V+zxmY7pwyIajOWteqoHHv+LnZc1WxwUntiBhxkyFdZiWduJf0K1aVbB6pPFXfuK29zAd2XEQ2VZXjv7odTTgEjUnGIfYTwxz9Vm7fencbZf9vi3SG8vSky6vZQhrBQRRIl95ZXwnKz49HIZxQ64jt6HECQdfkTmvCCUuUA3fqY5EfNgLF/Q8j1QsXeXII81vBTnJk7RYPtP49go1kmE/d9Av5kCqMgOQgis6DmW7KTHR2uqjfHNZOv2fA3GTwd5P68LVG4lxoVnSV4FkEN4ZRUF18s8BH6r4TCJ/4BogQOsBhb0nZxn0PSqK9Msec6+b4NGo1qIYlqNEF6n0dR+t6Lhsf979EpRQkk9ysfu3XWdjLzUjxhUvu8kTPOPcN8/hz1oOYTLh1g8HLf2XC7j8FuUIYCEPEpag=
+before_install:
+- echo "Before Install"
```

Comment:
Remove debug statements.

Reply:
Don’t we need to update secure hashes? Currently Travis is not failing because push is set up to run on master branch only.